src/modules/pv/pv_core.c

```diff
@@ -2335,6 +2335,110 @@ int pv_get_hfl(sip_msg_t *msg, pv_param_t *param, pv_value_t *res)
 		return pv_get_null(msg, param, res);
 	}
 
+	if((tv.flags == 0) && (tv.ri == HDR_PPI_T)) {
+		if(msg->ppi == NULL) {
+			LM_WARN("no PPI header\n");
+			return pv_get_null(msg, param, res);
+		}
+
+		if(parse_ppi_header(msg) < 0) {
+			LM_WARN("failed to parse PPI headers\n");
+			return pv_get_null(msg, param, res);
+		}
+
+		p_id_body_t *ppi_b = (p_id_body_t *)msg->ppi->parsed;
+		int ppi_header_count = 0;
+
```

Comment:
Declare the variables at the beginning of the function or at least at the beginning of the block, we try to keep compatibility with old C compilers as much as possible. Also, it makes it coherent with how the function code was written so far.

From coherence point of view of the function and file, it would be good to make the comments between `/* ... */`, not with `//`.

Reply:
Done.